src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertFromJsonCommand.cs

```diff
@@ -9,8 +9,8 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// The ConvertFrom-Json command
-    /// This command convert a Json string representation to a JsonObject
+    /// The ConvertFrom-Json command.
+    /// This command convert a Json string representation to a JsonObject.
```

Comment:
Use the plural 'converts' versus 'convert'

Reply:
Fixed.